planner/core/src/test/java/eu/seaclouds/platform/planner/core/PlannerTest.java

```diff
@@ -0,0 +1,90 @@
+package eu.seaclouds.platform.planner.core;
+
+import alien4cloud.tosca.parser.ParsingException;
+import com.google.common.io.Resources;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.*;
+
+
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@Test
+public class PlannerTest {
```

Comment:
The planner is also matchmaking technical requirements (which is something not performed by the Optimizer). If you like, we can extend the list of offerings with some offerings that are not matching tech reqs. 

Actually, there is also the discovery step (which we are omitting from testing, since we want a are testing with a static list of offerings, as you kindly suggested).